src/middlewared/middlewared/plugins/filesystem_/acl_linux.py

```diff
@@ -31,16 +31,31 @@ def acltool(self, path, action, uid, gid, options):
         if acltool.returncode != 0:
             raise CallError(f"acltool [{action}] on path {path} failed with error: [{acltool.stderr.decode().strip()}]")
 
-    def _common_perm_path_validate(self, schema, path, recursive, verrors):
+    def _common_perm_path_validate(self, schema, data, verrors):
+        is_cluster = data['path'].startswith('CLUSTER')
```

Comment:
other places specifically look for `CLUSTER:` (colon suffix)... will this be a problem ?

Reply:
I'll fix for consistency.